src/Main.java

```diff
@@ -1,6 +1,31 @@
 public class Main {
 
     public static void main(String[] args) {
+<<<<<<< Updated upstream
     System.out.println("hello world");
+=======
+
+    //Create a method that will check if a given string is a palindrome. If it is it should print “This string is a palindrome” if not print “This string is not a palindrome”
+        String original = "racecar";
```

Comment:
Your code works only with string which doesn’t contain symbols (%,&,<,?,*) or white spaces and only in lower case. Examples of multiple words polindromes https://examples.yourdictionary.com/palindrome-examples.html.